ding/worker/buffer/tests/test_benchmark.py

```diff
@@ -0,0 +1,130 @@
+import sys
+import timeit
+import torch
+import pytest
+import numpy as np
+
+# test different buffer size, eg: 1000, 10000, 100000;
+size_list = [1000, 10000]
+# test different tensor dim, eg: 32*32, 128*128, 512*512;
+data_dim_list = [32, 128]
+# repeat times.
+repeats = 100
+
+empty_base_setup = """import sys
```

Comment:
why use this format rather than normal function, can you provide some advantages of it in code comments

Reply:
What we test is not a complete function, but a small code chunk. I don't want to include the initialization processes, such as buffer creating and data preparation, in the test, but I didn't find a proper method to pass those initialized params in the beginning. While it is feasible through a new class, I rewrite it.